src/utils/ParseRouteUtils.js

```diff
@@ -261,6 +265,49 @@ function generateBoundingBox(path: Object): Object {
   };
 }
 
+/**
+ * Returns the distance between [startCoords] and [endCoords] in miles
+ * @param startCoords
+ * @param endCoords
+ * @returns {number}
+ */
+function getDistanceInMiles(startCoords: Object, endCoords: Object): number {
+  const lat1 = startCoords.lat;
+  const lon1 = startCoords.long;
+  const lat2 = endCoords.lat;
+  const lon2 = endCoords.long;
+  if (lat1 === lat2 && lon1 === lon2) {
+    return 0;
+  }
+
+  const radlat1 = Math.PI * lat1 / 180;
+  const radlat2 = Math.PI * lat2 / 180;
+  const theta = lon1 - lon2;
+  const radtheta = Math.PI * theta / 180;
+  let dist = Math.sin(radlat1) * Math.sin(radlat2) + Math.cos(radlat1) * Math.cos(radlat2) * Math.cos(radtheta);
+  if (dist > 1) {
+    dist = 1;
+  }
+  dist = Math.acos(dist);
+  dist = dist * 180 / Math.PI;
+  dist = dist * 60 * 1.1515;
```

Comment:
1.1515 needs a comment, and prefer all of these to be on one line with an explanation